### PR TITLE
add citation caution text

### PR DESCRIPTION
**Pull request by pgwillia**
screenshot from 2018-11-16 13-13-37
@seanluyk Is this what you were thinking?

### CODE REVIEW

**Comment by seanluyk** [comment]
@pgwillia, exactly, thanks! Just out of curiosity...how much longer did this take b/c of Blacklight?

**Comment by seanluyk** [comment]
Got it, thanks @pgwillia - it's helpful for me to know this

File: app/views/catalog/citation.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<p>Please verify the citations below with the appropriate style guide.</p>
```
Comment (murny) [inline app/views/catalog/citation.html.erb]: is this overriding the `citation.html.erb` view that comes from blacklight I assume? Or how/where does this `citiation.html.erb` gets rendered?

File: app/views/catalog/citation.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+<%= render :partial => 'citation' -%>
```
Comment (murny) [inline app/views/catalog/citation.html.erb]: Try not to use hash rockets when possible :sweat_smile: 

```suggestion
<%= render partial: 'citation' %>
```

Also there's a small difference between `%>` and `-%>`. The `-%>` will apparently trim the following line break (if the expression leaves a blank line) and there's questions if this actually works (as rails uses erubis to render ERB which might not support this)? Ultimately we tend not care about condensing our html and if we did we would minify it. So typically in Rails world, you hardly ever see the `-%>` or `<%-` usage.

File: app/views/catalog/citation.html.erb (hunk 1 above)
Comment (pgwillia) [reply app/views/catalog/citation.html.erb]: Correct.  This is the file in blacklight https://github.com/projectblacklight/blacklight/blob/v6.10.1/app/views/catalog/citation.html.erb

File: app/views/catalog/citation.html.erb (hunk 2 above)
Comment (pgwillia) [reply app/views/catalog/citation.html.erb]: You caught me.  Laziness.  Just copied from blacklight.

**Pull request merged**